src/styles/index.css

```diff
@@ -0,0 +1,69 @@
+/* @import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"); */
+
+@import "./reset.css";
+
+@import "./variables.css";
+
+@import "./blocks/card.css";
+
+@import "./blocks/stars.css";
+
+.imac {
+  width: 160px;
+  height: 134px;
+}
+
+.title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-dark);
+}
+
+.product-code {
+  color: #616070;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.price-text,
+
+.price-value {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.price-text {
```

Comment:
Correct the code according to this task and split your styles into 2 files `stars.css` `card.css`. This comment does not apply to the `variable` and `reset` files, they are correctly created
![image](https://user-images.githubusercontent.com/70009205/209989664-4c6155c8-75a6-4150-b9f9-0a85088f3f75.png)